web/src/composebox_typeahead.ts

```diff
@@ -1264,7 +1545,8 @@ export function initialize({on_enter_send}) {
     initialize_compose_typeahead("textarea#compose-textarea");
 
     $("#private_message_recipient").on("blur", function () {
-        const val = $(this).val();
+        const val = $("#private_message_recipient").val();
+        assert(typeof val === "string");
```

Comment:
Remember my [advice on `.val()` typing](https://chat.zulip.org/#narrow/stream/6-frontend/topic/TypeScript.20migration/near/1750154). However, something else is wrong here. `#private_message_recipient` is a `<div contenteditable>`, not an `<input>` or `<textarea>`, so it doesn’t make sense to call `.val()` (it will always give the empty string).

Reply:
Ah thanks for linking to this, I hadn't seen it before, though I think I've ended up doing the correct things several times anyways. Sorry for forgetting here.

It looks like this line is from [over 10 years ago](https://github.com/zulip/zulip/commit/09deef96117b8792081e4dc6b074aad605d0edfd#diff-f2b22549dc2fb4824b37e787d964a2c9f4f580e767f86b6d7ded1d0ff37d5a62R291) and probably before we had pills here. We can just remove this whole block.